Jenkinsfile

```diff
@@ -179,6 +209,10 @@ pipeline {
                 }
             }
         }
+
+
+
+        /*
```

Comment:
It looks like we can remove these comments and after merging this pr - test e2e these changes in release branch